Library/Homebrew/cmd/audit.rb

```diff
@@ -4,15 +4,18 @@
 #:
 #:    If no <formulae> are provided, all of them are checked.
 #:
-#:    If `--strict` is passed, additional checks are run. This should be used
-#:    when creating for new formulae.
+#:    If `--strict` is passed, additional checks are run, including Rubocop
+#:    style checks. This should be used when creating new formulae.
 #:
 #:    If `--online` is passed, additional slower checks that require a network
 #:    connection are run. This should be used when creating for new formulae.
 #:
 #:    `audit` exits with a non-zero status if any errors are found. This is useful,
 #:    for instance, for implementing pre-commit hooks.
 
+# Undocumented options:
```

Comment:
Surprisingly, no, because it takes a capital `D` instead of a lowercase `d` like `--debug`. I don't know why it's different, but it's a distinct option.